### PR TITLE
Updating DesiredUnsolicitedPongInterval for StreamWebSocketControl and MessageWebSocketControl

### PR DESCRIPTION
**Pull request by yyatmsft**

### CODE REVIEW

**Review by mikematteson — CHANGES_REQUESTED** [review]
@yyatmsft 
**Action required to pass PR acceptance checklist**

Thanks for your contribution. Our preliminary [PR acceptance checklist](https://review.docs.microsoft.com/en-us/windows-authoring-guide/managing-contributions/editorial-checklist?branch=master) includes the following localizability requirement:

* *Links to pages that do not require a locale code to render are coded as locale-agnostic (that is, do not contain "/en-us/" in the URL).*

Please remove the "/en-us" locale specifier from all link URLs in your submission, so that those links can go to content consistent with the user's preferred locale. Thanks!

Cc: @stevewhims

**Comment by yyatmsft** [comment]
@mikematteson I have made the change you requested. Please take another look. Thank you!

File: windows.networking.sockets/messagewebsocketcontrol_desiredunsolicitedponginterval.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ Allows an app to get and set the desired unsolicited WebSocket PONG interval.
 System.Timespan
 ## -remarks
 This terminology is defined in WebSocket RFC 6455, which is the time between unsolicited Pong control frames sent by the client to the server. The PONG serves as a unidirectional heartbeat and can be used for the client to determine network connection issues.  
+
+Note that setting the DesiredUnsolicitedPongInterval on a MessageWebSocket associated with a [ControlChannelTrigger](https://docs.microsoft.com/uwp/api/Windows.Networking.Sockets.ControlChannelTrigger) is not supported. Setting the property does not change the actual interval.
```
Comment (DavidGoll) [inline windows.networking.sockets/messagewebsocketcontrol_desiredunsolicitedponginterval.md]: Consider using Note formatting:
```
> [!NOTE]
> Setting the DesiredUnsolicitedPongInterval ...
```

**Review by Diego-Perez-Botero — APPROVED** [review]

**Review by mikematteson — APPROVED** [review]
@yyatmsft : Thanks for addressing the URLs.

@stevewhims : PR now **passes** checklist review. Please evaluate further and approve/merge if appropriate.

**Comment by yyatmsft** [comment]
Resolved @DavidGoll 's comment.

**Pull request merged**